### PR TITLE
feat: add missing properties to action type

### PR DESCRIPTION
**Pull request by ShermanE1**

### CODE REVIEW

**Review by mtruongsa — APPROVED** [review]

File: packages/context/src/objects/objects.ts [hunk 1]
```diff
@@ -216,34 +220,39 @@ export type Form = {
     entries?: FormEntry[];
 };
 
+export type ActionInputType =
+    | 'button'
```
Comment (actong) [inline packages/context/src/objects/objects.ts:224]: Is this supposed to be lower cased?  Also, what does this correspond to in a form?  Do we have buttons in forms?

Comment (ShermanE1) [reply packages/context/src/objects/objects.ts:224]: This corresponds to the submit/cancel buttons on a form. We append them to the form on the FE, so they don't ever get saved to the DB. 

We filter out these buttons from the form definition in some places on the FE once they are added in, so I added it here as we are using this type definition.

Comment (ShermanE1) [reply packages/context/src/objects/objects.ts:224]: Though if it is more accurate from a data perspective, I can append/override this type in the FE.

Comment (actong) [reply packages/context/src/objects/objects.ts:226]: What could do the button do?  Is it just the same as the submit/cancel buttons we have, or can they pretty much do anything with it?

Comment (ShermanE1) [reply packages/context/src/objects/objects.ts:226]: It seems like pretty much anything. FormIO expects an `onClick` handler.

**Review by actong — APPROVED** [review]

**Pull request merged**